.github/workflows/rust.yml

```diff
@@ -100,4 +100,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --all-features -- -D warnings
+          args: --all-features -- -D warnings -A clippy::absurd_extreme_comparisons
```

Comment:
better to add `#[allow(clippy::absurd_Extreme_comparisons)]` next to where the the warning is being given :o

Reply:
Yup, tried that but there were some issues with the `ensure!` macro where most absurd checks are being done. For some reason clippy ignored the decoration.